### DISCUSSION TITLE
BG-Remover V2.0

### DISCUSSION THREAD

**Discussion by developersharif** [developersharif/bgremover-app#1] [Q&A] [answered]
v2.0 Demo
https://user-images.githubusercontent.com/54396379/226102386-ede49d4e-7683-4777-be2e-104f3f335998.mp4

<hr /><em>This discussion was created from the release <a href='https://github.com/developersharif/bgremover-app/releases/tag/v2.0.0'>BG-Remover V2.0</a>.</em>

**Comment by Montu-Jasoliya** [comment]
html to dart

**Comment by dbucci69** [comment, marked as answer]
> v2.0 Demo https://user-images.githubusercontent.com/54396379/226102386-ede49d4e-7683-4777-be2e-104f3f335998.mp4
> _This discussion was created from the release [BG-Remover V2.0](https://github.com/developersharif/bgremover-app/releases/tag/v2.0.0)._

hello, thank you very much for you great tool! Is it possible to implement a CLI way to use it? 
bgremover.exe -i \<inputfileName\> -o \<outputfileName\>

and, obviously should save out image at same size of input image. Thank you.
Regards

**Reply by developersharif** [reply]
Yes, possible ,
currently i have not enough time for some feature improvement.
